code/modules/mob/living/silicon/pai/software.dm

```diff
@@ -4,24 +4,24 @@
 															//radiation eyes
 															//chem goggs
 															//mesons
-															"crew manifest" = 5,
-															"digital messenger" = 5,
-															"atmosphere sensor" = 5,
-															"photography module" = 5,
-															"camera zoom" = 10,
-															"printer module" = 10,
-															"remote signaler" = 10,
-															"medical records" = 10,
-															"security records" = 10,
-															"host scan" = 10,
-															"medical HUD" = 20,
-															"security HUD" = 20,
-															"loudness booster" = 20,
-															"newscaster" = 20,
-															"door jack" = 25,
-															"encryption keys" = 25,
-															"internal gps" = 35,
-															"universal translator" = 35
+															"Crew Manifest" = 5,
+															"Digital Messenger" = 5,
+															"Atmosphere Sensor" = 5,
+															"Photography Module" = 5,
+															"Camera Zoom" = 10,
+															"Printer Module" = 10,
```

Comment:
make all of the software names into DEFINEs please

Reply:
done